style.css

```diff
@@ -27,54 +27,56 @@ body {
 ================================*/
 form {
   background: #2c3e50;
-  height: 2.3em; 
+  height: 2.3em;
   width: 100%;
+  display: flex;
```

Comment:
`display: flex;` is "Flexbox", right...?
https://css-tricks.com/snippets/css/a-guide-to-flexbox/
(_I'm a complete **noob** at CSS ... so zero XP on this..._)
It _appears_ to be available in "_All Modern Browsers_" ... http://caniuse.com/#feat=flexbox
Given that this app uses `WebSockets` which is also not "_progressive enhancement_" http://caniuse.com/#search=websockets
I don't have an _issue_ with using Flexbox for this demo app. 
@iteles thoughts?